Jbarget/oxymore
# creates advertising page

### What changes have you made?

- created advertising page 
- reduced images to 720px 

(Designs approved by Aitor on 29/09, new high res image to follow but don't have a date for this)

### Which issue(s) does this PR fix?

Fixes #7 

### Screenshots (if there are design changes)
<img width="1440" alt="Screenshot 2020-09-25 at 18 44 24" src="https://user-images.githubusercontent.com/53219789/94293813-2bcc3280-ff5f-11ea-92a8-5f7276b12d9f.png">

![localhost_3000_advertising(iPhone 6_7_8)](https://user-images.githubusercontent.com/53219789/94294330-f411ba80-ff5f-11ea-892b-e6f277e95347.png)

### How to test
open the Advertising page and check:
- the images appear side by side on desktop with some text underneath
- the images and text appear as one column on mobile 
- the translation works on both texts
